lib/test-worker.js

```diff
@@ -1,38 +1,56 @@
 'use strict';
-/* eslint-disable import/order */
-const isObj = require('is-obj');
-const process = require('./process-adapter');
 
-const opts = process.opts;
-const testPath = opts.file;
+// Check if the test is being run without AVA cli
+{
+	/* eslint-disable import/order */
+	const path = require('path');
+	const chalk = require('chalk');
+
+	const isForked = typeof process.send === 'function';
```

Comment:
I am not so concerned about breaking `karma-ava` in the short term. But we need a strategy for where we will locate Node stuff that needs to be polyfilled for the browser. This is going to necessarily create some "oddness", as you will encounter the need to abstract out stuff you might prefer to inline throughout the codebase.